llvm/lib/Target/AArch64/AArch64InstrInfo.cpp

```diff
@@ -9694,19 +9694,26 @@ void AArch64InstrInfo::buildClearRegister(Register Reg, MachineBasicBlock &MBB,
                                           MachineBasicBlock::iterator Iter,
                                           DebugLoc &DL,
                                           bool AllowSideEffects) const {
-  const MachineFunction &MF = *MBB.getParent();
+  MachineFunction &MF = *MBB.getParent();
```

Comment:
Is this change necessary?

Reply:
No (I used the wrong function at first which was non-const)